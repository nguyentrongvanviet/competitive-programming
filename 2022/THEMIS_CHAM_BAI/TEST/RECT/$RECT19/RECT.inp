31
678059412 -144691146
-168042518 -64435452
809364971 709035459
-874399920 -112427321
114948485 -816366843
-580060534 543193894
32273288 -456687594
554603386 25756459
7330780 -391703837
217425512 242929900
88248239 -602229176
-24753078 -7468316
-264332760 -348356749
-874399920 -112427321
539611491 -708749138
-397027241 330759534
-663317919 854460027
809364971 709035459
217425512 242929900
304198692 -75359070
431993786 -284086132
-24753078 -7468316
-824797709 -668605843
932561207 -391796938
304198692 -75359070
-367626174 726227187
-517137277 -755106179
-57320618 80061093
-342137510 727249912
654081928 373039775
714375736 157222099
