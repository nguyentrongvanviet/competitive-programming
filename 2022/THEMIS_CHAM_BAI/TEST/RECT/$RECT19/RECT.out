191920239094423061